testdata/ppd-valid.json

```diff
@@ -34,34 +34,14 @@
                     "identificationNumber": "#83738AB#      ",
                     "individualName": "Steven Tander         ",
                     "discretionaryData": "  ",
-                    "addendaRecordIndicator": 1,
+                    "addendaRecordIndicator": 0,
                     "traceNumber": "121042880000001",
                     "category": "Forward"
                 }
-            ],
```

Comment:
Don't we need a batch control for the test?

Reply:
Did you see https://github.com/moov-io/ach/pull/388? It eliminates the need to include batch/file controls in the JSON. `ach` will try and tabulate the counts and sequence numbers then too. I hope it simplifies the API and doesn't introduce bugs. It's mentioned in the changelog, but I'm not calling it a "feature".